pip/basecommand.py

```diff
@@ -105,17 +105,9 @@ def parse_args(self, args):
     def main(self, args):
         options, args = self.parse_args(args)
 
-        if options.quiet:
-            if options.quiet == 1:
-                level = "WARNING"
-            elif options.quiet == 2:
-                level = "ERROR"
-            else:
-                level = "CRITICAL"
-        elif options.verbose:
-            level = "DEBUG"
-        else:
-            level = "INFO"
+        verbosity = options.verbose - options.quiet
+        level_dict = {1: "DEBUG", -1: "WARNING", -2: "ERROR", -3: "CRITICAL"}
```

Comment:
This isn't going to quite work because you don't want it to just be ``==``, but you really need the ``>`` and ``<`` behavior in the original snippet I posted. For example, this is going to do nothing if someone does ``pip install -qqqqqqqqqqqq``.